.gitlab-ci.yml

```diff
@@ -387,11 +387,14 @@ tests_ebpf:
   script:
     # For now only check bpf bytes since we don't have a way to run eBPF tests without mounting a debugfs
     - CGO_CFLAGS='-I/opt/libbcc/include' CGO_LDFLAGS='-Wl,-rpath,/opt/libbcc/lib -L/opt/libbcc/lib' inv -e system-probe.test --only-check-bpf-bytes
+    # Compile runtime security functional tests to be executed in kitchen tests
+    - inv -e security-agent.functional-tests --build-tags=ebpf_bindata --output=$DD_AGENT_TESTING_DIR/site-cookbooks/dd-security-agent-check/files/testsuite
```

Comment:
Should `--build-tags=ebpf_bindata` be implicit in the `security-agent.functional-tests` ? Can they be run without ebpf_bindata?

Reply:
Done